rdmd_test.d

```diff
@@ -56,13 +60,19 @@ void main(string[] args)
         "rdmd", &rdmd,
         "concurrency", &concurrencyTest,
         "m|model", &model,
+        "v|verbose", &verbose,
     );
 
     enforce(rdmd.exists, "Path to rdmd does not exist: %s".format(rdmd));
 
     rdmdApp = tempDir().buildPath("rdmd_app_") ~ binExt;
     if (rdmdApp.exists) std.file.remove(rdmdApp);
 
+    // compiler needs to be an absolute path becase
+    // we change directories
+    if(compiler.findAmong(dirSeparators))
```

Comment:
Space before (

Reply:
`compiler.canFind!isDirSeparator`. No need for `dirSeparators`.

Reply:
The `dirSeparators` is to support windows.  On windows directories can be separated by both forward slashes `/` and back slashes `\`.

Reply:
Look at the implementation of `isDirSeparator`. It already does that.

Reply:
Hmmm, I don't use std.algorithm too often, is there a function that can take advantage of `isDirSeparator` for this?
```
compiler.canFind!isDirSeparator()
```

Reply:
I don't understand your question. What's wrong with the expression you posted there (and I posted in my original comment)?

The line would then look like:

```d
if (compiler.canFind!isDirSeparator)
```

Reply:
Lol, I had no idea that would actually work, I just typed something that I though got my point accross and it just so happened to work :)